.github/actions/setup/action.yml

```diff
@@ -22,51 +22,179 @@ inputs:
   python-version:
     required: true
     type: string
+  cuda-version:
+    required: true
+    type: string
 
 runs:
   using: composite
   steps:
-    - name: Set REPO_DIR and Dump environment
+    # WAR: setup-python is not relocatable...
+    # see https://github.com/actions/setup-python/issues/871
+    - name: Set up Python ${{ inputs.python-version }}
+      if: ${{ startsWith(inputs.host-platform, 'linux') }}
+      id: setup-python
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.12"
+
+    # - name: Set up new PowerShell (pwsh)
+    #   if: ${{ startsWith(inputs.host-platform, 'win') }}
+    #   shell: powershell
+    #   env:
+    #     # doesn't seem there's an easy way to avoid hard-coding it?
+    #     PS_MSI_URL: https://github.com/PowerShell/PowerShell/releases/download/v7.4.6/PowerShell-7.4.6-win-x64.msi
+    #   run: |
+    #     Invoke-WebRequest -Uri "$env:PS_MSI_URL" -OutFile "ps_installer.msi"
+    #     Start-Process msiexec.exe -Wait -Verbose -ArgumentList '/i "ps_installer.msi" /qn REGISTER_MANIFEST=1'
+    #     echo "$env:ProgramFiles\\PowerShell\\7" >> $env:GITHUB_PATH
```

Comment:
We can clean this up now, as it's now tracked in https://github.com/nv-gha-runners/vm-images/issues/102. (Our images do not contain pre-installed `pwsh`.)

Reply:
removed